src/components/Functions/dataComponent.tsx

```diff
@@ -0,0 +1,26 @@
+import { ExclamationCircleIcon as ExclamationCircleIcon } from '@patternfly/react-icons';
+import React, { FC } from 'react';
+import { ChartDataComponentFunction } from './types';
+
+export enum ChartDataComponentNames {
+  scatterPlot = 'scatterPlot'
+}
+
+interface Props {
+  x?: number;
+  y?: number;
+  [key: string]: any;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const ExclamationCircleIconPoint: FC<Props> = ({ x, y, disableInlineStyles, ...props }) => {
```

Comment:
I think maybe we should not include react-icons in the library for this really specific example, this example could be moved to the example, and the library would not export any dataComponent as this is pretty specific. (Also I don't think that any of the helper function really belongs to the library, just keeping it there for backwards compatibility.)